src/kubelet.rs

```diff
@@ -320,7 +320,7 @@ pub trait Provider {
                             if let Some(cfkey) = env_src.field_ref {
                                 return fields
                                     .get(cfkey.field_path.as_str())
-                                    .and_then(|s| Some(s.to_string()))
+                                    .map(|s| s.to_string())
```

Comment:
I know it was `to_string` before, but can we turn this into `to_owned`?

Reply:
Oops, I meant to do that and forgot